ensembl_production/utils.py

```diff
@@ -27,14 +28,16 @@ def perl_string_to_python(s):
     else:
         return ''
 
+
 def perl_string_to_python_website(s):
```

Comment:
@marcoooo: Do we still need this method?

Reply:
I'll double check because, I think I use it to convert the data somewhere in the migration step. Thanks for spotting it.

Reply:
We need Well we don't use it in the code, but only in the migration steps. I must say I have to come up with a solution about that, since this is supposed to be deleted. May be in another PR where we'll remove completely those steps